contracts/discrepancies/nonce/InternalCaller.sol

```diff
@@ -28,26 +30,29 @@ contract InternalCaller {
     }
 
     function callRevertWithRevertReason(address _addr) external {
-        _addr.call(abi.encodeWithSignature("revertWithRevertReason()"));
+        (bool success,) = _addr.call(abi.encodeWithSignature("revertWithRevertReason()"));
+        require(success);
     }
 
     function callRevertWithoutRevertReason(address _addr) external {
-        _addr.call(abi.encodeWithSignature("revertWithoutRevertReason()"));
+        (bool success,) = _addr.call(abi.encodeWithSignature("revertWithoutRevertReason()"));
+        require(success);
     }
 
     function sendTo(address payable _addr) external {
-        _addr.send(1);
```

Comment:
Q: is this a replacement due to warning?
Is there a way to ignore the warning and maintain the coverage?
Maybe even add support for the `transfer` also

Reply:
Yes it's due to the warning but I read it too quickly and replaced it with `transfer` but it indeed requires its returned var to be used. Pushed in the fix!